### PR TITLE
Add configurable eye separation and focal length for VR

### PR DESCRIPTION
**Pull request by bagnell**
Adds `Scene.eyeSeparation` and `Scene.focalLength` properties to configure VR settings.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
@bagnell, thanks for the pull request! Maintainers, we have a signed CLA from @bagnell, so you can review this at any time.

I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).



__I am a bot who helps you make Cesium awesome!__ Thanks again.

**Comment by bagnell** [comment]
Fixes #3909.

**Comment by lilleyse** [comment]
Looks good.

File: Source/Scene/Scene.js [hunk 1]
```diff
@@ -653,6 +653,18 @@ define([
         this._actualInvertClassificationColor = Color.clone(this._invertClassificationColor);
         this._invertClassification = new InvertClassification();
 
+        /**
+         * The focal length for use when with cardboard or WebVR.
```
Comment (pjcozzi) [inline Source/Scene/Scene.js]: Please add the units and the default value.

File: Source/Scene/Scene.js [hunk 2]
```diff
@@ -653,6 +653,18 @@ define([
...
+         * @type {Number}
+         */
+        this.focalLength = undefined;
+
+        /**
+         * The eye separation distance in meters for use with cardboard or WebVR.
```
Comment (pjcozzi) [inline Source/Scene/Scene.js]: Add the default value.

File: Source/Scene/Scene.js [hunk 3]
```diff
@@ -2282,8 +2294,8 @@ define([
             var savedCamera = Camera.clone(camera, scene._cameraVR);
 
             var near = camera.frustum.near;
-            var fo = near * 5.0;
-            var eyeSeparation = fo / 30.0;
+            var fo = near * defaultValue(scene.focalLength, 5.0);
```
Comment (pjcozzi) [inline Source/Scene/Scene.js]: Are you sure this should be done here and not at initialization like Cesium does everywhere else?  Is it valid to set this back to `undefined`?

**Comment by pjcozzi** [comment]
Is it possible to add reasonable unit tests?

**Pull request merged**